great-idea/index.html

```diff
@@ -16,7 +16,78 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <div class="container">
+
+      <header>
+        <nav>
+          <a href="#">Services</a>
```

Comment:
Your page looks great except it's sort of hard to navigate between pages. To solve this adding services.html in place of the # should get you there.